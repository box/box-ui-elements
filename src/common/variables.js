// @flow
const BOX_BLUE = '#0061d5';
const SEESEE = '#ccc';
const RESIN_TAG_TARGET = 'data-resin-target';

export { BOX_BLUE, SEESEE, RESIN_TAG_TARGET };
