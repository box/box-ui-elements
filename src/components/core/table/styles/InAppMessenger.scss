$banner-height: 40px;

/** When a body-button-banner template is showing in banner frame, the banner-body-button-banner class
 * will be added to InAppMessengerProvider component, the selector will use this to generate different
 * CSS props. Here the class added to InAppMessengerProvider is <frameName>-<templateName>, ex:
 * For body-button-banner, frameName is banner, templateName is body-button-banner.
 */
$in-app-messenger-banner-selector: '.in-app-messenger-provider.banner-body-button-banner';

@mixin in-app-messenger-banner-mixin($top) {
    top: $top + $banner-height;
}

@mixin in-app-messenger-banner-padding-top-mixin($padding-top) {
    padding-top: $padding-top + $banner-height;
}
