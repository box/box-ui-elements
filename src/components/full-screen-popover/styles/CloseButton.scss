@import '../../../styles/variables';
@import './variables';

.btn.bdl-close-btn {
    width: $close-btn-size;
    height: $close-btn-size;
    padding: unset;
    background-color: $bdl-gray-10;
    border: unset;
}
