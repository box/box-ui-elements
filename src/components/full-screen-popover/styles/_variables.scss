$overlay-header-height: 64px;

$close-btn-size: 32px;
