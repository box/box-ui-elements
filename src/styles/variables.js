// @flow
// File auto-generated

export const bdlBoxBlue = '#0061d5'; // bdl-box-blue
export const bdlSecondaryBlue = '#003c84'; // bdl-secondary-blue
export const bdlLightBlue = '#2486fc'; // bdl-light-blue
export const bdlGreenLight = '#26c281'; // bdl-green-light
export const bdlYellorange = '#f5b31b'; // bdl-yellorange
export const bdlYellow = '#ffd700'; // bdl-yellow
export const bdlGrimace = '#4826c2'; // bdl-grimace
export const bdlPurpleRain = '#9f3fed'; // bdl-purple-rain
export const bdlWatermelonRed = '#ed3757'; // bdl-watermelon-red
export const bdlYellow10 = '#fffbe5'; // bdl-yellow-10
export const bdlBoxBlue50 = '#619de5'; // bdl-box-blue-50
export const bdlBoxBlue05 = '#f2f7fd'; // bdl-box-blue-05
export const bdlNeutral01 = '#222'; // bdl-neutral-01
export const bdlNeutral02 = '#767676'; // bdl-neutral-02
export const bdlNeutral03 = '#909090'; // bdl-neutral-03
export const bdlNeutral04 = '#eaeaea'; // bdl-neutral-04
export const bdlNeutral05 = '#f3f3f3'; // bdl-neutral-05
export const bdlNeutral06 = '#fafafa'; // bdl-neutral-06
export const boxBlue = '#0061d5'; // box-blue
export const primaryColor = '#0061d5'; // primary-color
export const korayIng = '#2a62b9'; // koray-ing
export const pictonBlue = '#3392ec'; // picton-blue
export const white = '#fff'; // white
export const black = '#000'; // black
export const hoverBlueBackground = '#f6fafd'; // hover-blue-background
export const hoverBlueBorder = '#d9e3e9'; // hover-blue-border
export const selectedBlueBackground = '#ecf4fc'; // selected-blue-background
export const selectedBlueBorder = '#bfd4df'; // selected-blue-border
export const babyBlue = '#bed4de'; // baby-blue
export const bluePurple = '#6d83ff'; // blue-purple
export const hoverBlue = '#48b6f3'; // hover-blue
export const dustyPlum = '#9b59b6'; // dusty-plum
export const sadSunjin = '#72388f'; // sad-sunjin
export const paleMint = '#eefbf5'; // pale-mint
export const thatsMint = '#26c281'; // thats-mint
export const shadyAcres = '#15aa61'; // shady-acres
export const wildRice = '#ede097'; // wild-rice
export const earlyDawn = '#fffbe5'; // early-dawn
export const lemonMeringue = '#ffd522'; // lemon-meringue
export const dehydrated = '#f5b31b'; // dehydrated
export const kumera = '#81721b'; // kumera
export const madMax = '#f39c12'; // mad-max
export const ayersRock = '#e06e2b'; // ayers-rock
export const amaranth = '#ed3757'; // amaranth
export const lavenderBlush = '#feeff2'; // lavender-blush
export const carissma = '#eb8b9c'; // carissma
export const greatBallsOfFire = '#ef4836'; // great-balls-of-fire
export const kingCrimson = '#c82341'; // king-crimson
export const bfive = '#b5b5b5'; // bfive
export const dtwo = '#d2d2d2'; // dtwo
export const efive = '#e5e5e5'; // efive
export const ffive = '#f5f5f5'; // ffive
export const fnine = '#f9f9f9'; // fnine
export const seesee = '#ccc'; // seesee
export const haze = '#fcfcfc'; // haze
export const offWhite = '#eaeaea'; // off-white
export const sfFog = '#bdc3c7'; // sf-fog
export const karlFog = '#fbfbfc'; // karl-fog
export const silverChalice = '#afafaf'; // silver-chalice
export const stormsAComin = '#737f8b'; // storms-a-comin
export const blueSteel = '#2c3e50'; // blue-steel
export const betterBlack = '#494949'; // better-black
export const downtownGrey = '#999ea4'; // downtown-grey
export const twos = '#222'; // twos
export const fours = '#444'; // fours
export const sevens = '#777'; // sevens
export const eights = '#888'; // eights
export const nines = '#999'; // nines
export const eees = '#eee'; // eees
export const tendemobGrey = '#64686d'; // tendemob-grey
export const sunsetGrey = '#464a4f'; // sunset-grey
export const seventySixers = '#767676'; // seventy-sixers
export const approxMischkaGrey = '#a3adb9'; // approx-mischka-grey
export const lightestCharcoal = '#c4c4c4'; // lightest-charcoal
export const purpleRain = '#9f3fed'; // purple-rain
export const modalDialogZIndex = '160'; // modal-dialog-z-index
export const menuZIndex = '170'; // menu-z-index
export const notificationsWrapperZIndex = '180'; // notifications-wrapper-z-index
export const overlayZIndex = '190'; // overlay-z-index
export const tooltipZIndex = '190'; // tooltip-z-index
export const bdlLineHeight = '20px'; // bdl-line-height
export const bdlBorderRadiusSize = '4px'; // bdl-border-radius-size
