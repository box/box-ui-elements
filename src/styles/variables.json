{
    "white": "#fff",
    "black": "#000",
    "countBadgeBackground": "#e9163c",
    "bdl-box-blue": "#0061d5",
    "bdl-box-blue-80": "#3381dd",
    "bdl-box-blue-50": "#7fb0ea",
    "bdl-box-blue-40": "#99c0ee",
    "bdl-box-blue-30": "#b2cff2",
    "bdl-box-blue-20": "#ccdff7",
    "bdl-box-blue-10": "#e5effa",
    "bdl-box-blue-05": "#f2f7fd",
    "bdl-box-blue-02": "#fafcfe",
    "bdl-gray": "#222",
    "bdl-gray-80": "#4e4e4e",
    "bdl-gray-62": "#767676",
    "bdl-gray-50": "#909090",
    "bdl-gray-40": "#a7a7a7",
    "bdl-gray-30": "#bcbcbc",
    "bdl-gray-20": "#d3d3d3",
    "bdl-gray-10": "#e8e8e8",
    "bdl-gray-05": "#f4f4f4",
    "bdl-gray-02": "#fbfbfb",
    "bdl-dark-blue": "#003c84",
    "bdl-dark-blue-50": "#7f9dc1",
    "bdl-dark-blue-20": "#ccd8e6",
    "bdl-dark-blue-10": "#e5ebf2",
    "bdl-dark-blue-05": "#f2f5f9",
    "bdl-dark-blue-02": "#fafbfd",
    "bdl-light-blue": "#2486fc",
    "bdl-light-blue-50": "#91c2fd",
    "bdl-light-blue-20": "#d3e7fe",
    "bdl-light-blue-10": "#e9f2fe",
    "bdl-light-blue-05": "#f4f9ff",
    "bdl-light-blue-02": "#fbfdff",
    "bdl-yellorange": "#f5b31b",
    "bdl-yellorange-50": "#fad98d",
    "bdl-yellorange-20": "#fdf0d1",
    "bdl-yellorange-10": "#fef7e8",
    "bdl-yellorange-05": "#fefbf3",
    "bdl-yellorange-02": "#fffefb",
    "bdl-yellow": "#ffd700",
    "bdl-yellow-50": "#ffeb7f",
    "bdl-yellow-20": "#fff7cc",
    "bdl-yellow-10": "#fffbe5",
    "bdl-yellow-05": "#fffdf2",
    "bdl-yellow-02": "#fffefa",
    "bdl-green-light": "#26c281",
    "bdl-green-light-50": "#92e0c0",
    "bdl-green-light-20": "#d4f3e6",
    "bdl-green-light-10": "#e9f8f2",
    "bdl-green-light-05": "#f4fcf8",
    "bdl-green-light-02": "#fbfefd",
    "bdl-grimace": "#4826c2",
    "bdl-grimace-50": "#a392e0",
    "bdl-grimace-20": "#dad4f3",
    "bdl-grimace-10": "#ece9f8",
    "bdl-grimace-05": "#f6f4fc",
    "bdl-grimace-02": "#fbfbfe",
    "bdl-orange": "#f1811a",
    "bdl-orange-50": "#f8c08c",
    "bdl-orange-20": "#fce6d1",
    "bdl-orange-10": "#fdf2e8",
    "bdl-orange-05": "#fef8f3",
    "bdl-orange-02": "#fffdfb",
    "bdl-purple-rain": "#9f3fed",
    "bdl-purple-rain-50": "#cf9ff6",
    "bdl-purple-rain-20": "#ecd9fb",
    "bdl-purple-rain-10": "#f5ebfd",
    "bdl-purple-rain-05": "#faf5fe",
    "bdl-purple-rain-02": "#fdfbff",
    "bdl-watermelon-red": "#ed3757",
    "bdl-watermelon-red-50": "#f69bab",
    "bdl-watermelon-red-20": "#fbd7dd",
    "bdl-watermelon-red-10": "#fdebee",
    "bdl-watermelon-red-05": "#fef5f6",
    "bdl-watermelon-red-02": "#fffbfc",
    "bdl-line-height": "20px",
    "bdl-fontSize": "13px",
    "bdl-fontSize-minimum": "11px",
    "bdl-fontSize--dejaBlue": "14px",
    "avatar-colors": ["#0061d5", "#003c84", "#767676", "#222", "#4826c2", "#9f3fed"],
    "modal-dialog-z-index": "160",
    "menu-z-index": "170",
    "notifications-wrapper-z-index": "180",
    "overlay-z-index": "190",
    "tooltip-z-index": "190",
    "bdl-border-radius-size": "4px",
    "bdl-border-radius-size-med": "6px",
    "bdl-border-radius-size-large": "8px",
    "bdl-border-radius-size-xlarge": "12px",
    "bdl-grid-unit": "4px",
    "bdl-grid-unit-px": "4",
    "bdl-btn-border-width": "1px",
    "bdl-btn-height": "32px",
    "bdl-btn-height-large": "40px",
    "bdl-btn-icon-size": "20px",
    "bdl-btn-text-icon-size": "16px",
    "bdl-btn-padding-horizontal": "16px",
    "bdl-transitionDurationBase": "0.15s",
    "bdl-timingFunction": ["cubic-bezier(0", "0", "0.6", "1)"]
}
