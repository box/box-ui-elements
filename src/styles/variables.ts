/* File auto-generated */
export const white = '#fff'; // white
export const black = '#000'; // black
export const countBadgeBackground = '#e9163c'; // countBadgeBackground
export const bdlBoxBlue = '#0061d5'; // bdl-box-blue
export const bdlBoxBlue80 = '#3381dd'; // bdl-box-blue-80
export const bdlBoxBlue50 = '#7fb0ea'; // bdl-box-blue-50
export const bdlBoxBlue40 = '#99c0ee'; // bdl-box-blue-40
export const bdlBoxBlue30 = '#b2cff2'; // bdl-box-blue-30
export const bdlBoxBlue20 = '#ccdff7'; // bdl-box-blue-20
export const bdlBoxBlue10 = '#e5effa'; // bdl-box-blue-10
export const bdlBoxBlue05 = '#f2f7fd'; // bdl-box-blue-05
export const bdlBoxBlue02 = '#fafcfe'; // bdl-box-blue-02
export const bdlGray = '#222'; // bdl-gray
export const bdlGray80 = '#4e4e4e'; // bdl-gray-80
export const bdlGray62 = '#767676'; // bdl-gray-62
export const bdlGray50 = '#909090'; // bdl-gray-50
export const bdlGray40 = '#a7a7a7'; // bdl-gray-40
export const bdlGray30 = '#bcbcbc'; // bdl-gray-30
export const bdlGray20 = '#d3d3d3'; // bdl-gray-20
export const bdlGray10 = '#e8e8e8'; // bdl-gray-10
export const bdlGray05 = '#f4f4f4'; // bdl-gray-05
export const bdlGray02 = '#fbfbfb'; // bdl-gray-02
export const bdlDarkBlue = '#003c84'; // bdl-dark-blue
export const bdlDarkBlue50 = '#7f9dc1'; // bdl-dark-blue-50
export const bdlDarkBlue20 = '#ccd8e6'; // bdl-dark-blue-20
export const bdlDarkBlue10 = '#e5ebf2'; // bdl-dark-blue-10
export const bdlDarkBlue05 = '#f2f5f9'; // bdl-dark-blue-05
export const bdlDarkBlue02 = '#fafbfd'; // bdl-dark-blue-02
export const bdlLightBlue = '#2486fc'; // bdl-light-blue
export const bdlLightBlue50 = '#91c2fd'; // bdl-light-blue-50
export const bdlLightBlue20 = '#d3e7fe'; // bdl-light-blue-20
export const bdlLightBlue10 = '#e9f2fe'; // bdl-light-blue-10
export const bdlLightBlue05 = '#f4f9ff'; // bdl-light-blue-05
export const bdlLightBlue02 = '#fbfdff'; // bdl-light-blue-02
export const bdlYellorange = '#f5b31b'; // bdl-yellorange
export const bdlYellorange50 = '#fad98d'; // bdl-yellorange-50
export const bdlYellorange20 = '#fdf0d1'; // bdl-yellorange-20
export const bdlYellorange10 = '#fef7e8'; // bdl-yellorange-10
export const bdlYellorange05 = '#fefbf3'; // bdl-yellorange-05
export const bdlYellorange02 = '#fffefb'; // bdl-yellorange-02
export const bdlYellow = '#ffd700'; // bdl-yellow
export const bdlYellow50 = '#ffeb7f'; // bdl-yellow-50
export const bdlYellow20 = '#fff7cc'; // bdl-yellow-20
export const bdlYellow10 = '#fffbe5'; // bdl-yellow-10
export const bdlYellow05 = '#fffdf2'; // bdl-yellow-05
export const bdlYellow02 = '#fffefa'; // bdl-yellow-02
export const bdlGreenLight = '#26c281'; // bdl-green-light
export const bdlGreenLight50 = '#92e0c0'; // bdl-green-light-50
export const bdlGreenLight20 = '#d4f3e6'; // bdl-green-light-20
export const bdlGreenLight10 = '#e9f8f2'; // bdl-green-light-10
export const bdlGreenLight05 = '#f4fcf8'; // bdl-green-light-05
export const bdlGreenLight02 = '#fbfefd'; // bdl-green-light-02
export const bdlGrimace = '#4826c2'; // bdl-grimace
export const bdlGrimace50 = '#a392e0'; // bdl-grimace-50
export const bdlGrimace20 = '#dad4f3'; // bdl-grimace-20
export const bdlGrimace10 = '#ece9f8'; // bdl-grimace-10
export const bdlGrimace05 = '#f6f4fc'; // bdl-grimace-05
export const bdlGrimace02 = '#fbfbfe'; // bdl-grimace-02
export const bdlOrange = '#f1811a'; // bdl-orange
export const bdlOrange50 = '#f8c08c'; // bdl-orange-50
export const bdlOrange20 = '#fce6d1'; // bdl-orange-20
export const bdlOrange10 = '#fdf2e8'; // bdl-orange-10
export const bdlOrange05 = '#fef8f3'; // bdl-orange-05
export const bdlOrange02 = '#fffdfb'; // bdl-orange-02
export const bdlPurpleRain = '#9f3fed'; // bdl-purple-rain
export const bdlPurpleRain50 = '#cf9ff6'; // bdl-purple-rain-50
export const bdlPurpleRain20 = '#ecd9fb'; // bdl-purple-rain-20
export const bdlPurpleRain10 = '#f5ebfd'; // bdl-purple-rain-10
export const bdlPurpleRain05 = '#faf5fe'; // bdl-purple-rain-05
export const bdlPurpleRain02 = '#fdfbff'; // bdl-purple-rain-02
export const bdlWatermelonRed = '#ed3757'; // bdl-watermelon-red
export const bdlWatermelonRed50 = '#f69bab'; // bdl-watermelon-red-50
export const bdlWatermelonRed20 = '#fbd7dd'; // bdl-watermelon-red-20
export const bdlWatermelonRed10 = '#fdebee'; // bdl-watermelon-red-10
export const bdlWatermelonRed05 = '#fef5f6'; // bdl-watermelon-red-05
export const bdlWatermelonRed02 = '#fffbfc'; // bdl-watermelon-red-02
export const bdlLineHeight = '20px'; // bdl-line-height
export const bdlFontSize = '13px'; // bdl-fontSize
export const bdlFontSizeMinimum = '11px'; // bdl-fontSize-minimum
export const bdlFontSizeDejaBlue = '14px'; // bdl-fontSize--dejaBlue
export const avatarColors = ['#0061d5', '#003c84', '#767676', '#222', '#4826c2', '#9f3fed']; // avatar-colors
export const modalDialogZIndex = '160'; // modal-dialog-z-index
export const menuZIndex = '170'; // menu-z-index
export const notificationsWrapperZIndex = '180'; // notifications-wrapper-z-index
export const overlayZIndex = '190'; // overlay-z-index
export const tooltipZIndex = '190'; // tooltip-z-index
export const bdlBorderRadiusSize = '4px'; // bdl-border-radius-size
export const bdlBorderRadiusSizeMed = '6px'; // bdl-border-radius-size-med
export const bdlBorderRadiusSizeLarge = '8px'; // bdl-border-radius-size-large
export const bdlBorderRadiusSizeXlarge = '12px'; // bdl-border-radius-size-xlarge
export const bdlGridUnit = '4px'; // bdl-grid-unit
export const bdlGridUnitPx = '4'; // bdl-grid-unit-px
export const bdlBtnBorderWidth = '1px'; // bdl-btn-border-width
export const bdlBtnHeight = '32px'; // bdl-btn-height
export const bdlBtnHeightLarge = '40px'; // bdl-btn-height-large
export const bdlBtnIconSize = '20px'; // bdl-btn-icon-size
export const bdlBtnTextIconSize = '16px'; // bdl-btn-text-icon-size
export const bdlBtnPaddingHorizontal = '16px'; // bdl-btn-padding-horizontal
export const bdlTransitionDurationBase = '0.15s'; // bdl-transitionDurationBase
export const bdlTimingFunction = ['cubic-bezier(0', '0', '0.6', '1)']; // bdl-timingFunction
