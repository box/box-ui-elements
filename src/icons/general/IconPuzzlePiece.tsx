import * as React from 'react';
import classNames from 'classnames';
import AccessibleSVG from '../accessible-svg';
import { bdlGray40 } from '../../styles/variables';

import { Icon } from '../iconTypes';

const IconPuzzlePiece = ({ className = '', color = bdlGray40, height = 32, title, width = 32 }: Icon) => (
    <AccessibleSVG
        className={classNames('bdl-IconPuzzlePiece', className)}
        height={height}
        title={title}
        viewBox="0 0 14 14"
        width={width}
    >
        <path
            d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
            fill={color}
        />
    </AccessibleSVG>
);

export default IconPuzzlePiece;
