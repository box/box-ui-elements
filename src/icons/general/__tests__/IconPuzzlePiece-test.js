import * as React from 'react';

import IconPuzzlePiece from '../IconPuzzlePiece';
import { silverChalice } from '../../../styles/variables';

describe('icons/general/IconPuzzlePiece', () => {
    test('should correctly render default icon with default color', () => {
        const dimension = 14;
        const wrapper = shallow(<IconPuzzlePiece />);

        expect(wrapper.hasClass('bdl-IconPuzzlePiece')).toEqual(true);
        expect(wrapper.find('path').prop('fill')).toEqual(silverChalice);
        expect(wrapper.find('AccessibleSVG').prop('width')).toEqual(dimension);
        expect(wrapper.find('AccessibleSVG').prop('height')).toEqual(dimension);
        expect(wrapper.find('AccessibleSVG').prop('viewBox')).toEqual(`0 0 ${dimension} ${dimension}`);
    });

    test('should correctly render icon with specified color', () => {
        const color = '#fcfcfc';
        const wrapper = shallow(<IconPuzzlePiece color={color} />);

        expect(wrapper.find('path').prop('fill')).toEqual(color);
    });

    test('should correctly render icon with specified width, height, and viewBox value', () => {
        const dimension = 16;
        const wrapper = shallow(<IconPuzzlePiece dimension={dimension} />);

        expect(wrapper.find('AccessibleSVG').prop('width')).toEqual(dimension);
        expect(wrapper.find('AccessibleSVG').prop('height')).toEqual(dimension);
        expect(wrapper.find('AccessibleSVG').prop('viewBox')).toEqual(`0 0 ${dimension} ${dimension}`);
    });

    test('should correctly render icon with title', () => {
        const title = 'abcde';
        const wrapper = shallow(<IconPuzzlePiece title={title} />);

        expect(wrapper.find('AccessibleSVG').prop('title')).toEqual(title);
    });
});
