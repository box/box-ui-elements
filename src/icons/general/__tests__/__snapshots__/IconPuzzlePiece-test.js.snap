// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons/general/IconPuzzlePiece should correctly render default icon with default color 1`] = `
<AccessibleSVG
  className="bdl-IconPuzzlePiece"
  height={32}
  viewBox="0 0 14 14"
  width={32}
>
  <path
    d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
    fill="#a7a7a7"
  />
</AccessibleSVG>
`;

exports[`icons/general/IconPuzzlePiece should correctly render icon with custom class name 1`] = `
<AccessibleSVG
  className="bdl-IconPuzzlePiece rectangular"
  height={32}
  viewBox="0 0 14 14"
  width={32}
>
  <path
    d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
    fill="#a7a7a7"
  />
</AccessibleSVG>
`;

exports[`icons/general/IconPuzzlePiece should correctly render icon with specified color 1`] = `
<AccessibleSVG
  className="bdl-IconPuzzlePiece"
  height={32}
  viewBox="0 0 14 14"
  width={32}
>
  <path
    d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
    fill="#fcfcfc"
  />
</AccessibleSVG>
`;

exports[`icons/general/IconPuzzlePiece should correctly render icon with specified width and height and default viewBox value 1`] = `
<AccessibleSVG
  className="bdl-IconPuzzlePiece"
  height={16}
  viewBox="0 0 14 14"
  width={16}
>
  <path
    d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
    fill="#a7a7a7"
  />
</AccessibleSVG>
`;

exports[`icons/general/IconPuzzlePiece should correctly render icon with title 1`] = `
<AccessibleSVG
  className="bdl-IconPuzzlePiece"
  height={32}
  title="abcde"
  viewBox="0 0 14 14"
  width={32}
>
  <path
    d="M0 4.16v3c.17.85.65.93 1.15.8a1.72 1.72 0 0 1 1.31-.61 1.7 1.7 0 0 1 0 3.4 1.72 1.72 0 0 1-1.31-.61c-.5-.13-1-.05-1.15.8v3h9.84V11c.17-.85.64-.93 1.15-.8A1.7 1.7 0 1 0 11 8c-.51.13-1 0-1.15-.8v-3h-3C6 4 5.88 3.52 6 3a1.7 1.7 0 1 0-2.17 0c.13.51 0 1-.8 1.15z"
    fill="#a7a7a7"
  />
</AccessibleSVG>
`;
